contracts/ArrakisV2.sol

```diff
@@ -249,44 +249,48 @@ contract ArrakisV2 is IUniswapV3MintCallback, ArrakisV2Storage {
 
         {
             Withdraw memory aggregator;
-            for (uint256 i; i < rebalanceParams_.removes.length; i++) {
+            for (uint256 i; i < rebalanceParams_.burns.length; i++) {
                 IUniswapV3Pool pool = IUniswapV3Pool(
                     mFactory.getPool(
                         address(mToken0),
                         address(mToken1),
-                        rebalanceParams_.removes[i].range.feeTier
+                        rebalanceParams_.burns[i].range.feeTier
                     )
                 );
 
                 uint128 liquidity = Position.getLiquidityByRange(
                     pool,
                     address(this),
-                    rebalanceParams_.removes[i].range.lowerTick,
-                    rebalanceParams_.removes[i].range.upperTick
+                    rebalanceParams_.burns[i].range.lowerTick,
+                    rebalanceParams_.burns[i].range.upperTick
                 );
 
                 if (liquidity == 0) continue;
 
                 uint128 liquidityToWithdraw;
 
-                if (rebalanceParams_.removes[i].liquidity == type(uint128).max)
-                    liquidityToWithdraw = liquidity;
-                else
-                    liquidityToWithdraw = rebalanceParams_.removes[i].liquidity;
+                if (rebalanceParams_.burns[i].liquidity == type(uint128).max)
```

Comment:
why not to use line 261 liquidity variable here?

Reply:
fixed